fedireads/migrations/0042_sitesettings.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.0.3 on 2020-06-01 18:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fedireads', '0041_user_remote_id'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SiteSettings',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(default='wyrms.cthulahoops.org', max_length=100)),
+                ('instance_description', models.TextField(default='This instance has no description.')),
+                ('code_of_conduct', models.TextField(default='Add a code of conduct here.')),
+                ('allow_registration', models.BooleanField(default=True)),
```

Comment:
can we add an optional logo field here as well?

Reply:
Sure! Storing image data in the database?

I figure there's lots more settings that can be added to this, so this is mainly to get the structure in.

Reply:
I think that works -- thanks so much for building out this feature!

Reply:
I might add the logo in separate PR - there's enough in this already.

You're welcome. It's fun to do a feature again after all the hidden auth work!

Reply:
no need to prioritize logo, it just occurred to me

Reply:
Cool! Though it might be fun to work out how to store the image data.